pkg/build/cmd/reaper.go

```diff
@@ -41,33 +41,33 @@ func (reaper *BuildConfigReaper) Stop(namespace, name string, timeout time.Durat
 	var bcPotentialBuilds []buildapi.Build
 
 	// Collect builds related to the config.
-	builds, err := reaper.oc.Builds(namespace).List(metav1.ListOptions{LabelSelector: buildutil.BuildConfigSelector(name).String()})
+	builds, err := reaper.buildClient.Build().Builds(namespace).List(metav1.ListOptions{LabelSelector: buildutil.BuildConfigSelector(name).String()})
 	if err != nil {
 		return err
 	}
 
 	bcPotentialBuilds = append(bcPotentialBuilds, builds.Items...)
 
 	// Collect deprecated builds related to the config.
-	// TODO: Delete this block after BuildConfigLabelDeprecated is removed.
-	builds, err = reaper.oc.Builds(namespace).List(metav1.ListOptions{LabelSelector: buildutil.BuildConfigSelectorDeprecated(name).String()})
+	// TODO: Delete this blbuildClientk after BuildConfigLabelDeprecated is removed.
+	builds, err = reaper.buildClient.Build().Builds(namespace).List(metav1.ListOptions{LabelSelector: buildutil.BuildConfigSelectorDeprecated(name).String()})
 	if err != nil {
 		return err
 	}
 
 	bcPotentialBuilds = append(bcPotentialBuilds, builds.Items...)
 
-	// A map of builds associated with this build configuration
+	// A map of builds assbuildClientiated with this build configuration
 	bcBuilds := make(map[ktypes.UID]buildapi.Build)
 
 	// Because of name length limits in the BuildConfigSelector, annotations are used to ensure
-	// reliable selection of associated builds.
+	// reliable selection of assbuildClientiated builds.
```

Comment:
bad search/replace? :)  (couple of spots)

Reply:
yeah... my sed is weak